### PR TITLE
Add Signalgo for Bridge Deployer

### PR DESCRIPTION
**Pull request by kubo6472**

### CODE REVIEW

File: cmd/bbctl/config.go [hunk 1]
```diff
@@ -130,6 +130,7 @@ var bridgeIPSuffix = map[string]string{
 	"googlechat": "20",
 	"twitter":    "27",
 	"signal":     "28",
+	"signalgo":   "28",
```
Comment (kubo6472) [inline cmd/bbctl/config.go:133]: This might be very wrong, let me know if a fix is needed, please

**Comment by kubo6472** [comment]
It's not [here](https://mau.dev/-/snippets/8).

**Comment by tulir** [comment]
It's already supported, it's called `signal`

**Comment by kubo6472** [comment]
Oh, so signal will deploy signalgo at this time? Sorry then :D

**Pull request closed without merging**